File: baml_language/crates/baml_tests/benches/cross_lang/workloads/python/array_iter_10k.py
def build_array() -> list[int]:
    arr: list[int] = []
    for i in range(10000):
        arr.append(i)
    return arr


def main() -> int:
    arr = build_array()
    s = 0
    for x in arr:
        s += x
    return s

File: baml_language/crates/baml_tests/benches/cross_lang/workloads/go/array_iter_10k.go
package crosslang

func buildArray() []int {
	arr := []int{}
	for i := 0; i < 10000; i++ {
		arr = append(arr, i)
	}
	return arr
}

func ArrayIter10K() int {
	arr := buildArray()
	s := 0
	for _, x := range arr {
		s += x
	}
	return s
}
